packages/block-editor/src/hooks/border.js

```diff
@@ -42,22 +62,73 @@ export function BorderPanel( props ) {
 		return null;
 	}
 
+	const defaultBorderControls = getBlockSupport( props.name, [
+		BORDER_SUPPORT_KEY,
+		'__experimentalDefaultControls',
+	] );
+
+	// Callback to reset all block support attributes controlled via this panel.
+	const resetAll = () => {
+		const { style } = props.attributes;
+		const newStyle = cleanEmptyObject( {
+			...style,
+			border: undefined,
+		} );
+
+		props.setAttributes( { style: newStyle } );
```

Comment:
```suggestion
		props.setAttributes( { style: newStyle, borderColor: undefined } );
```

Reply:
I'd checked that `borderColor` was removed from the attributes in the code view of the editor. It is, despite missing the attribute being reset here. I believe that is a result of adjusting border colors when the width changes. 

Thanks for catching this one!